pkg/Microsoft.NETFramework.Compatibility.Shims/Microsoft.NETFramework.Compatibility.Shims.builds

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <!-- only build during the AllConfigurations leg because it depends on other nupkgs from this leg -->
```

Comment:
NIT: do we even use this file at all? I was checking packages.builds and looks like the way we build this package is by building the pkgproj directly, so I don't think that we need the builds file, but I can be mistaken.